.changeset/cyan-dolphins-change.md

```diff
@@ -0,0 +1,29 @@
+---
+'xstate': minor
+---
+
+Tags can now be added to state node configs under the `.tags` property:
+
+```js
+const machine = createMachine({
+  initial: 'green',
+  states: {
+    green: {
+      tags: 'go' // single tag
+    },
+    yellow: {
+      tags: 'go'
+    },
+    red: {
+      tags: ['stop', 'other'] // multiple tags
+    }
+  }
+});
+```
+
+You can query whether a state has a tag via `state.hasTag(tag)`:
+
+```js
+const canGo = state.hasTag('go');
+// => `true` if in 'green' or 'red' state
```

Comment:
Hi!, maybe I don't understand how tags works, but shouldn't this example be:

```diff
-// => `true` if in 'green' or 'red' state
+// => `true` if in 'green' or 'yellow' state
```

Reply:
Oops, just updated it!